sources/packages/api/src/route-controllers/application/application.student.controller.ts

```diff
@@ -91,14 +92,23 @@ export class ApplicationController extends BaseController {
     private readonly sfasPartTimeApplicationsService: SFASPartTimeApplicationsService,
     private readonly configService: ConfigService,
     private readonly disbursementScheduleService: DisbursementScheduleService,
+    private readonly locationService: InstitutionLocationService,
+    private readonly programService: EducationProgramService,
     private readonly assessmentService: StudentAssessmentService,
   ) {
     super();
     this.config = this.configService.getConfig();
   }
 
-  @AllowAuthorizedParty(AuthorizedParties.student)
   @Get(":id")
+  @ApiResponse({
```

Comment:
Can it be changed to use the specific _@ApiOkResponse_?
Same for the below one 😉

Reply:
😉 missed it. Thanks